dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/OpentracingUserFeatureAccessService.java

```diff
@@ -38,9 +38,22 @@
 
     private static OpentracingTracerFactory opentracingTracerFactory;
 
+    private static boolean factoryFirstUse = true;
+
     @Reference
     public void setOpentracingTracerFactory(OpentracingTracerFactory opentracingTracerFactory) {
         OpentracingUserFeatureAccessService.opentracingTracerFactory = opentracingTracerFactory;
+        factoryFirstUse = false;
+
+        if (opentracingTracerFactory != null) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled()) {
```

Comment:
I think Alasdair's comment was to not log a message for successful configuration of an OpenTracing provider.  Seeing that a class name from the user feature would be logged rather than the provider name makes me agree with that even more so.  The best we should be doing here is to trace it, however, assuming autogenerated trace is enabled, the entry point for this method already covers that, so can just delete this block.